azure-communication-ui/demo-app/src/chat/AndroidManifest.xml

```diff
@@ -32,6 +32,7 @@
             android:exported="true"
             android:label="@string/azure_communication_ui_chat_demo_app_title"
             android:theme="@style/Launcher.Theme"
+            android:windowSoftInputMode="adjustResize"
```

Comment:
https://github.com/Azure/communication-ui-library-android/blob/main/docs/CHANGELOG_UI_CHAT.md
Lets update the change log and specify its a fix to the demo app.